.github/workflows/android_build.yml

```diff
@@ -97,9 +101,13 @@ jobs:
     - name: 'Install dependencies'
       if: steps.should_run.outputs.run_ci_job == 'true'
       run: cd mobile && ./ci/mac_ci_setup.sh --android
-    - name: 'Start simulator'
+    - uses: nick-fields/retry@943e742917ac94714d2f408a0e8320f2d1fcafcd
       if: steps.should_run.outputs.run_ci_job == 'true'
-      run: cd mobile && ./ci/mac_start_emulator.sh
+      name: 'Start simulator'
```

Comment:
If it's not too much trouble, mind changing this from "simulator" to "emulator" like you did on line 61? (Here and the two others below)

Reply:
Oops, thanks for catching the ones I missed.